src/main/java/duke/data/exception/DukeException.java

```diff
@@ -0,0 +1,10 @@
+package duke.data.exception;
+
+/**
+ * Exception class for errors related to Duke
+ */
+public class DukeException extends Exception {
```

Comment:
I am a bit confused as to why the exception class is stored under the data folder?